apollo-router/src/plugins/external.rs

```diff
@@ -57,13 +210,24 @@ struct BaseConf {
     sdl: bool,
 }
 
+/// What information is passed to a subgraph request/response stage
 #[derive(Clone, Debug, Default, Deserialize, PartialEq, Serialize, JsonSchema)]
 #[serde(default)]
-struct RouterStage {
-    /// The request configuration
-    request: Option<BaseConf>,
-    /// The response configuration
-    response: Option<BaseConf>,
+struct SubgraphConf {
+    /// Send the headers
+    headers: bool,
+    /// Send the context
+    context: bool,
+    /// Send the body
+    body: bool,
+    /// Send the service name
+    service: bool,
+    /// Send the subgraph URI
+    /// TODO: This might not make sense at the response layer,
```

Comment:
These docs will be part of the json schema, so probably shouldn't have TODOs in.

Reply:
Good catch, removed in a8d57597